src/views/Home.js

```diff
@@ -222,33 +221,21 @@ function Home() {
                                             <Tab className={classes.tab} label="Plugin" {...a11yProps(2)} />
                                         </Tabs>
                                         <TabPanel className={classes.tab_panel} value={tabValue} index={0}>
-                                            <Typography component={'span'} className={classes.info}>Schema Name: {curr_zone['host_access_control_config']['schema_name']}</Typography>
-                                            <Typography component={'span'} className={classes.info}>Schema Version: {curr_zone['host_access_control_config']['schema_version']}</Typography>
-                                            <Typography component={'span'} className={classes.info}>Catalog Schema Version: {curr_zone['version']['catalog_schema_version']}</Typography>
-                                            <Typography component={'span'} className={classes.info}>Configuration Schema Version: {curr_zone['version']['configuration_schema_version']}</Typography>
-                                            <Typography component={'span'} className={classes.info}>iRODS Version: {curr_zone['version']['irods_version']}</Typography>
-                                            <Typography component={'span'} className={classes.info}>Installation Time: {curr_zone['version']['installation_time']}</Typography>
+                                            <Typography component={'span'} className={classes.info}>Schema Name: {zoneContext['icat_server']['host_access_control_config']['schema_name']}</Typography>
```

Comment:
know that we'll change the zone_report in 4.3.0 to use 'provider' and 'consumer' rather than 'icat_server'.

sorry ahead of time.  :)